app/client/ts/video-model.ts

```diff
@@ -276,7 +276,7 @@ export default class Video implements Drop {
         const comment = await VideoComment.new(
             owner,
             text,
-            new Date().getTime(),
+            new Date(),
```

Comment:
As of now, it just saves us a couple of lines on the watch page in case we need to get the month, day, etc. of when the comment was posted.